presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1292,4 +1293,17 @@ public HiveClientConfig setTemporaryTableStorageFormat(HiveStorageFormat tempora
         this.temporaryTableStorageFormat = temporaryTableStorageFormat;
         return this;
     }
+
+    public boolean isUseRewindableSplitSource()
+    {
+        return useRewindableSplitSource;
+    }
+
+    @Config("hive.use-rewindable-split-source")
+    @ConfigDescription("Use rewindable hive split source and preload splits for grouped execution")
```

Comment:
> and preload splits for grouped execution

I would either drop that part, or spell it like `(preloads splits for grouped execution)`